book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] this is a many-to-many relationship which should be set up a little different than a has-many / belongs-to relationship, as you can see in [this example](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) you should create a new instance of Rental and pass the date, person and the book as arguments. (make sure to notice how the book is passed)